static/app/components/events/interfaces/threads/threadSelector/selectedOption.tsx

```diff
@@ -12,12 +12,18 @@ type Props = {
 type ThreadInfo = {
   filename?: string;
   label?: string;
+  state?: string | null;
```

Comment:
Really small nit, but I noticed that we have a lot these state types.
Would it make sense to reuse the state type from the `Thread` interface instead since it looks to be the same type? like this:
```suggestion
  state: Thread['state'];
```
Plus and import somewhere above:
```
import {Thread} from 'sentry/types';
```